src/integrations/prefect-dbt/prefect_dbt/cli/commands.py

```diff
@@ -148,16 +148,48 @@ def trigger_dbt_cli_command_flow():
         )
 
     # append the options
-    command += f" --profiles-dir {profiles_dir}"
+    cli_args = [command]
+    cli_args.append("--profiles-dir")
+    cli_args.append(profiles_dir)
     if project_dir is not None:
         project_dir = Path(project_dir).expanduser()
-        command += f" --project-dir {project_dir}"
+        cli_args.append("--project-dir")
+        cli_args.append(project_dir)
 
-    # fix up empty shell_run_command_kwargs
-    shell_run_command_kwargs = shell_run_command_kwargs or {}
+    if command_kwargs:
+        cli_args.append(command_kwargs)
 
-    logger.info(f"Running dbt command: {command}")
-    result = await shell_run_command.fn(command=command, **shell_run_command_kwargs)
+    # fix up empty shell_run_command_kwargs
+    dbt_runner_client = dbtRunner()
+    logger.info(f"Running dbt command: {cli_args}")
+    result: dbtRunnerResult = dbt_runner_client.invoke(cli_args)
+
+    if result.exception is not None:
+        logger.info(result)
```

Comment:
This looks like a debug log that was left in since there's an error log directly below it?
```suggestion
```